GiTracker/ViewModels/IssueListViewModel.cs

```diff
@@ -1,20 +1,73 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using System.Collections.ObjectModel;
 using System.Threading.Tasks;
+using GiTracker.Models;
 using GiTracker.Services.Api;
+using Prism.Commands;
+using Prism.Navigation;
 
 namespace GiTracker.ViewModels
 {
     public class IssueListViewModel : BaseViewModel
     {
         readonly IGitApiService _gitApiService;
+        readonly INavigationService _navigationService;
 
-        public IssueListViewModel(IGitApiServiceFactory gitApiServiceFactory)
+        public IssueListViewModel(IGitApiServiceFactory gitApiServiceFactory,
+            INavigationService navigationService)
         {
             _gitApiService = gitApiServiceFactory.GetApiService();
+            _navigationService = navigationService;
         }
 
+        public override void OnNavigatedTo(NavigationParameters parameters)
+        {
+            base.OnNavigatedTo(parameters);
+
+            UpdateIssues();
```

Comment:
UpdateIssues() is awaiting a Task inside itself and will contain error handling later